packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/backup/backup.constants.js

```diff
@@ -102,52 +102,79 @@ export const BACKUP_CONDITIONS_URL = {
 };
 
 export const BACKUP_TARIFF_URL = {
```

Comment:
As it is located in a component maybe we can instead provide the urls depending on the use case

Reply:
Makes sense. Thanks for the suggestion. This is done :)